layers/+lang/extra-langs/packages.el

```diff
@@ -36,8 +36,8 @@
 (defun extra-langs/init-thrift ()
   (use-package thrift :defer t))
 
-;; no associated extension because conflicts with more common Objective-C, manually invoke for .m files.
```

Comment:
modify this comment, but don't delete it entirely. The info about .m files is important